src/app/(views)/github/t/@pullButtons/pulls/page.tsx

```diff
@@ -29,6 +38,62 @@ export default function Page() {
     openAllRepo()
   }
 
+  const handleSelectAllDependabotClick = () => {
+    if (repos) {
+      repos.map((repo) => {
```

Comment:
I believe you don't know the difference between `map` and `forEach`. Check it out.

Reply:
No,, No believe!! I know the difference between map and forEach!! TT😢